website-builder/Dockerfile

```diff
@@ -1,38 +1,35 @@
-FROM ruby:2.4
- 
+FROM ruby:2.7-alpine
+
 LABEL maintainer="Debezium Community"
- 
-ENV SITE_HOME=/site \
-    CACHE_HOME=/cache \
-    NODE_PATH=/usr/local/share/.config/yarn/global/node_modules
 
-# Install Node.js - Required by Antora
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
-    && apt-get install -y nodejs
+ENV SITE_HOME=/site
 
-# Install Yarn
-RUN npm install -g yarn
+RUN apk add --no-cache build-base gcc bash cmake git jq nodejs yarn
 
 # Install Antora framework
-RUN yarn global add @antora/cli@2.3.4 @antora/site-generator-default@2.3.4 \
+RUN yarn global add @antora/cli @antora/site-generator-default \
```

Comment:
Can we keep the versions? Always better to explicitly specify them, so to not suffer from any breaking updates.

Reply:
Added it back. Yes, It works locally. I tested with `uidoyen/newjekyll`.